doc/input-output-reference/src/overview/group-airflow-network.tex

```diff
@@ -997,6 +1000,46 @@ \subsubsection{Inputs}\label{inputs-3-003}
   0.65;                 !- Air mass flow exponent {dimensionless}
 \end{lstlisting}
 
+\subsection{AirflowNetwork:MultiZone:SpecifiedFlowRate}\label{airflownetworkmultizonespecifiedflow}
+
+The specified flow object is used to specify flow through a surface. The object has three fields
+and will cause the mass flow $\dot m$ to be set appropriately as 
+
+\begin{equation}
+\dot m = F
+\end{equation}
+
+when the mass flow rate $F$ is specified or as 
+
+\begin{equation}
+\dot m = \rho Q
+\end{equation}
+
+when the volume flow rate $Q$ is specified.
+
+\subsubsection{Inputs}\label{inputs-sf}
+
+\paragraph{Field: Name}\label{field-sf-name}
+
+This is a name for this object. It is referenced by an \hyperref[airflownetworkmultizonesurface]{AirflowNetwork:MultiZone:Surface} object.
+
+\paragraph{Field: Air Flow Value}\label{field-sf-air-flow-value}
+
+This numeric field is used to input the flow rate $F$ or $Q$. The units of the value are determined by the following field.
+
+\paragraph{Field: Discharge Coefficient}\label{field-sf-air-flow-units}
+
+This choice field determines the units of the air flow value. Two options are available, ``kg/s'' and ``m3/s''.
+
+An IDF example is shown below:
+
+\begin{lstlisting}
+AirflowNetwork:MultiZone:SpecifiedFlowRate,
+  SpecMassFlow,         !- Name of specified flow component
+  0.0001,               !- Air flow rate
+  kg/s;                 !- Air flow units
+\end{lstlisting}
+
```

Comment:
It is better to provide description for flow direction. The positive value show floe from thermal zone to outdoor....

Reply:
@lgu1234 Good point, will add discussion of that as well.